pages/settings/index.vue

```diff
@@ -7,6 +7,7 @@ definePageMeta({
 
 <template>
   <AccountInformation />
+  <SubscriptionSettings />
```

Comment:
Should be in separated menu Settings / Subscriptions. 
Also user can buy more than one subscription.
Also user should be able to change the plan (Regular / Premium).